### DISCUSSION TITLE
show download speed

### DISCUSSION THREAD

**Discussion by SCIEN-00** [microsoft/winget-cli#4020] [Ideas] [answered]
I didn't find any topic on that so I thought to write it here. Recently I updated some packages and wondered why it took so long, good indicator would be to see download speed next to progress bar.
Something like that perhaps:
![image](https://github.com/microsoft/winget-cli/assets/11405851/f1d7ff2f-d3fc-4b6c-b6a5-18b61b5a1c00)

**Comment by mdanish-kh** [comment, marked as answer]
Duplicate of 
- https://github.com/microsoft/winget-cli/issues/596

You can add a👍 to that issue to raise its priority
